apps/documentation/src/app/using-smart-ng-rx/web-sockets/index.md

```diff
@@ -0,0 +1,100 @@
+# WebSockets
+
+To use SmartNgRX with WebSockets, you'll need to establish your own WebSocket connection. The only information the that you'll need to pass up to the client are which type of action needs to be performed, the table name that is impacted, and the ids of the records that are impacted.
+
+## Updates
+
+So, as an example, if you update the content of a row in the `users` table, you'll need to send a message to the client that might like this:
```

Comment:
**suggestion (documentation):** Correct 'might like this' to 'might look like this'